src/lando/main/models/uplift.py

```diff
@@ -0,0 +1,141 @@
+import json
+from typing import Any
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from lando.main.models import BaseModel
+
+# Yes/No constants for re-use in `TextChoices`, since `Enum`
+# can't be subclassed.
+YES = "yes", "Yes"
+NO = "no", "No"
+
+
+class YesNoChoices(models.TextChoices):
+    """A yes/no choice selection."""
+
+    YES = YES
+    NO = NO
+
+
+class YesNoUnknownChoices(models.TextChoices):
+    """A yes/no/unknown choice selection."""
+
+    YES = YES
+    NO = NO
+    UNKNOWN = "unknown", "Unknown"
+
+
+class LowMediumHighChoices(models.TextChoices):
+    """A low/medium/high choice selection."""
+
+    LOW = "low", "Low"
+    MEDIUM = "medium", "Medium"
+    HIGH = "high", "High"
+
+
+class UpliftAssessment(BaseModel):
+    """Represents the responses to the uplift request form."""
+
+    # User who submitted the form.
+    user = models.ForeignKey(User, on_delete=models.DO_NOTHING)
+
+    # User impact if declined?
+    user_impact = models.TextField(blank=False)
+
+    # Code covered by automated testing?
+    covered_by_testing = models.CharField(
```

Comment:
Curious if there was a reason why this and others were not set up as BooleanField (with and without null=True, e.g., for "unknown")?